mysql-data/ib_buffer_pool

```diff
@@ -0,0 +1,176 @@
+4,3
```

Comment:
Suggestion: Do not include  `mysql-data/` folder. It is heavy and include big binary files.
To start a docker with some content in the startup, you can use a single SQL file and create a volume in folder `/docker-entrypoint-initdb.d` as described here: https://hub.docker.com/_/mysql (Initializing a fresh instance Section)

Reply:
example; https://github.com/rafaelsq/boiler/blob/master/Makefile#L25-L29